src/cloudflare/internal/vectorize-api.ts

```diff
@@ -217,8 +239,13 @@ async function toJson<T = unknown>(response: Response): Promise<T> {
 export function makeBinding(env: {
   fetcher: Fetcher;
   indexId: string;
-}): VectorizeIndex {
-  return new VectorizeIndexImpl(env.fetcher, env.indexId);
+  indexVersion?: VectorizeVersion;
+}): Vectorize {
+  return new VectorizeIndexImpl(
+    env.fetcher,
+    env.indexId,
+    env.indexVersion ?? "v1"
```

Comment:
This ensures backwards compatibility, since previously built pipelines won't have these inner globals.
We default to v1 if is it not present.